src/sentry/event_consumer.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import, print_function
+
+from sentry.coreapi import Auth
+from sentry.event_manager import EventManager
+from sentry.web.api import process_event
+
+
+def process_event_from_kafka(message):
```

Comment:
Let me be clear that this PR does not move processing to a Kafka consumer. This code exists to test that the message published to Kafka provides enough data to pick up where we left off and finish the job.

It will eventually be wrapped in a real Kafka consumer, but we'll be doing a lot of testing before this actually takes over production work.